content/guides/docker-concepts/the-basics/what-is-a-registry.md

```diff
@@ -21,10 +21,34 @@ While Docker Hub is a popular option, there are many other available container r
 
 ### Registry vs. repository
 
-While you're working with registries, you might hear the terms **registry** and **repository** as if they're interchangeable. Even though they're related, they're not quite the same thing.
+While you're working with registries, you might hear the terms _registry_ and _repository_ as if they're interchangeable. Even though they're related, they're not quite the same thing.
 
 A _registry_ is a centralized location that stores and manages container images, whereas a _repository_ is a collection of related container images within a registry. Think of it as a folder where you organize your images based on projects. Each repository contains one or more container images.
 
+The following diagram shows the relationship between a registry, repositories, and images.
+
+```console
+    +-----------------------------------+
+    |             Registry              |
+    |-----------------------------------|
+    |                                   |
+    |  +-----------------------------+  |
+    |  |        Repository A         |  |
+    |  |-----------------------------|  |
+    |  | - Image: project-a:v1.0     |  |
+    |  | - Image: project-a:v2.0     |  |
+    |  +-----------------------------+  |
+    |                                   |
+    |  +-----------------------------+  |
+    |  |        Repository B         |  |
+    |  |-----------------------------|  |
+    |  | - Image: project-b:v1.0     |  |
+    |  | - Image: project-b:v1.1     |  |
+    |  | - Image: project-b:v2.0     |  |
+    |  +-----------------------------+  |
+    +-----------------------------------+
+```
```

Comment:
I'm a sucker for ascii diagrams. In fact, Hugo supports a diagram syntax called GoAT natively. I tried it out with this PR and it looks something like this (with slight modifications)

<img width="632" alt="image" src="https://github.com/docker/docs/assets/35727626/927371e4-b388-49cc-bc97-b3f938126e39">

Here's the diff if you want to try it out (but feel free to merge as-is, it doesn't really add anything, just a fun experiment)

````suggestion
```goat {class="text-sm"}
+---------------------------------------+
|               Registry                |
|---------------------------------------|
|                                       |
|    +-----------------------------+    |
|    |        Repository A         |    |
|    |-----------------------------|    |
|    |   Image: project-a:v1.0     |    |
|    |   Image: project-a:v2.0     |    |
|    +-----------------------------+    |
|                                       |
|    +-----------------------------+    |
|    |        Repository B         |    |
|    |-----------------------------|    |
|    |   Image: project-b:v1.0     |    |
|    |   Image: project-b:v1.1     |    |
|    |   Image: project-b:v2.0     |    |
|    +-----------------------------+    |
|                                       |
+---------------------------------------+
```
````